### PR TITLE
[ADDED] staging model for condition and intermediate model adding

### PR DESCRIPTION
**Pull request by lpanda2**
## Describe your changes
- added the johnston classification columns to the condition table on principal discharge diagnosis.
- added a sample summary table to get feedback
- filtered for ed visits in the original table

## Issues
- https://github.com/tuva-health/ed_classification/issues/3
- https://github.com/tuva-health/ed_classification/issues/4

## How has this been tested?
_Please describe the tests you ran to verify your changes.  Provide instructions or code to reproduce output._
- checked sum counts on the medicare lds five percent data to validate that the numbers are correct

## Reviewer focus
_Please summarize the specific items you’d like the reviewer(s) to look into._
- please look at the ed_summary table for final output
- please share feedback on structure, organization and documentation

## Checklist before requesting a review
- [x]  (Optional) I have recorded a Loom performing a self-review of my code
- [x]  My code follows style guidelines
- [x]  I have commented my code as necessary
- [ ]  (New models only) I have implemented generic dbt tests to validate primary keys/uniqueness in my model
- [ ]  I have added at least one Github label to this PR

## Loom link
https://www.loom.com/share/3ffe9d9ac6e44ab7818da09f68e4ebaf

### CODE REVIEW

File: models/_models.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
\ No newline at end of file
+      - name: classification_order
+      - name: classification_name
+      - name: condition_row_ct
+      - name: encounter_ct
+      - name: claim_ct
+      - name: patient_ct
+      - name: claim_paid_amount_sum
+      - name: pct_claim_row_ct
+      - name: pct_claim_paid_amount_sum
```
Comment (sarah-tuva) [inline models/_models.yml:18]: What does PCT stand for? For final models, we generally prefer longer/more descriptive column names. 

Do you mind if I update permissions in your dev database to see how this table looks with data? I received an insufficient privilege error.

File: models/intermediate/ed_classification__johnston_conditions_with_class.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
+-- Filter conditions to those that were classified and pick the classification
+-- with the greatest probability
```
Comment (sarah-tuva) [inline models/intermediate/ed_classification__johnston_conditions_with_class.sql:2]: Do you plan to leave these descriptions in the SQL? I do like that they are describing the logic. If they are going to stay do you mind using multi-line quotes here `/* */`?

File: models/intermediate/ed_classification__johnston_conditions_with_class.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+
+-- {{ config(enabled=var('ed_classification_enabled',var('tuva_packages_enabled',True))) }}
+
+
+{{
+  config(
+     materialized="table",
+  )
+}}
```
Comment (sarah-tuva) [inline models/intermediate/ed_classification__johnston_conditions_with_class.sql:12]: We like to have all config options in one place in the _models.yml file that way, it's easier to make changes when needed. The exception is the enabled config. There seems to be a bug with dbt, and those have to stay in the SQL files.

File: models/_models.yml (hunk 1 above)
Comment (lpanda2) [reply models/_models.yml:18]: absolutely -- also pct = percent but i'm happy to use your preferred naming convention.

File: models/intermediate/ed_classification__johnston_conditions_with_class.sql (hunk 3 above)
Comment (lpanda2) [reply models/intermediate/ed_classification__johnston_conditions_with_class.sql:12]: sounds good, i'll go ahead and make those changes. is there a contribution guide i could look at?

Comment (sarah-tuva) [reply models/intermediate/ed_classification__johnston_conditions_with_class.sql]: @lpanda2 Yep! I added a link to the guide in this [issue](https://github.com/tuva-health/ed_classification/issues/9). I thought we could walk through it together today if we have time during our check-in.

**Comment by lpanda2** [comment]
remove condition_type = discharge diagnosis

File: models/_models.yml (hunk 1 above)
Comment (lpanda2) [reply models/_models.yml:18]: will handle in this https://github.com/tuva-health/ed_classification/issues/11

**Review by sarah-tuva — APPROVED** [review]

**Pull request merged**